packages/plugin-core/src/commands/OpenLink.ts

```diff
@@ -79,3 +86,25 @@ export class OpenLinkCommand extends BasicCommand<CommandOpts, CommandOutput> {
     return { filepath: assetPath };
   }
 }
+
+const showDepreciationWarnign = () => {
+  AnalyticsUtils.track(ExtensionEvents.DeprecationNoticeShow, {
+    source: DENDRON_COMMANDS.OPEN_LINK.key,
+  });
+  window
+    .showWarningMessage(
+      "Heads up that OpenLink is being deprecated and will be replaced with the 'Goto' command",
+      "See whats changed"
+    )
+    .then((resp) => {
+      if (resp === "See whats changed") {
+        AnalyticsUtils.track(ExtensionEvents.DeprecationNoticeAccept, {
+          source: DENDRON_COMMANDS.OPEN_LINK.key,
+        });
+        VSCodeUtils.openLink(
+          //TODO: This link needs to be updated
+          "https://wiki.dendron.so/notes/ftohqknticu6bw4cfmzskq6"
```

Comment:
we don't actually need a separate link since theres no action to take.
can we just display the following warning?
> Open link will be deprecated. Please use `Dendron: Go to` command instead